openhexa/sdk/pipelines/runtime.py

```diff
@@ -19,6 +45,96 @@ def import_pipeline(pipeline_dir_path: str):
     return pipeline
 
 
+def _get_openhexa_decorator_id(tree: ast.AST, decorator: str):
+    imported_nodes = [node for node in tree.body if isinstance(node, ast.Import) or isinstance(node, ast.ImportFrom)]
```

Comment:
I don't think this line is necessary since what you do below is filtering on them anyway.